src/Hosting/SystemdServices/Directory.Build.props

```diff
@@ -0,0 +1,8 @@
+<Project>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)..\, Directory.Build.props))\Directory.Build.props" />
+
+  <PropertyGroup>
+    <!-- Source build does not support Systemd. -->
```

Comment:
Hmm, there's no extra dependencies that force this. Why not?

Reply:
It's not in a shared framework right now, so it doesn't **have** to be source-built. Having said that, it is a component for linux services which means it is probably desirable to land it in some kind of shared framework some day. Since it has no extra deps, I'd agree, let's try to put this in source-build now to avoid issues later.

Reply:
I copied this over from the WindowsService, I'll remove it.